src/index.ts

```diff
@@ -53,6 +53,8 @@ export const typeStat = async (argv: TypeStatArgv, output: ProcessOutput): Promi
     }
 
     output.stdout(chalk.greenBright("🚀 Welcome to TypeStat!"));
+    output.stdout(chalk.yellowBright("⚠️ TypeStat is still very early stage and experimental. ⚠️ While it will improve your code, it will likely add syntax and type errors. Use TypeStat as a starting point before you manually fix and verify any changes."))
```

Comment:
Ah, I'd been hoping to have these on separate lines. Users have a bad habit of not reading anything that's too wide or overflows.

From https://github.com/JoshuaKGoldberg/TypeStat/issues/1046#issue-1330184550:

> ```plaintext
> ⚠️ TypeStat is still very early stage and experimental. ⚠️
> While it will improve your code, it will likely add syntax and type errors.
> Use TypeStat as a starting point before you manually fix and verify any changes.
```

Reply:
Doh! Initially had it split but I was not sure if the text would auto-wrap or not.